Runtime/Core/Submesh.cs

```diff
@@ -186,6 +186,8 @@ public static Submesh[] GetSubmeshes(IEnumerable<Face> faces, int submeshCount,
                 }
             }
 
+            //Removing submeshes that does not contain vertices
+            submeshes = submeshes.Where(submesh => submesh.m_Indexes.Length != 0).ToArray();
```

Comment:
I expect that this will be an edge case, so if possible let's avoid allocating a second collection.

It also occurs to me that this could potentially remap face materials - if for example submesh 0 and 2 are valid, but submesh 1 is not, then this will incorrectly remap submesh 2 faces to material 1.